suite-common/wallet-config/src/types.ts

```diff
@@ -0,0 +1,7 @@
+import { networks, NetworkSymbol } from './networksConfig';
+
+export type TokenSymbol = string & { __type: 'TokenSymbol' };
+
+export const isNetworkSymbol = (
```

Comment:
This code is used in `suite-common/wallet-core/src/blockchain/blockchainThunks.ts`. It's not shown in the diff because it was using different function with same name which I removed. I think this is better but if you find it useless, I can revert this change.

For the mixing. In suite desktop, in some components networkSymbol and string (for erc20 token "symbol") is used intechangebly with some type casting which is not ideal and this function could be helpful in the future for determining between these 2 don't you think?

Reply:
You can keep it function here but please don't modify it's origin type signature. We will solve this issue when it will be needed, it may be useful but maybe we will do with different solution so it will be forgotten here.

Reply:
I removed the EhereumTokenSymbol but I think it makes sense to keep NetworkSymbol type as the name `isNetworkSymbol` suggests (previous implementation expected string)

Reply:
https://github.com/trezor/trezor-suite/pull/7676/commits/4961fc7a9a7d0a7573924322467aabdd9d7e8b05